client/components/resource-details/Reviews.tsx

```diff
@@ -146,7 +146,8 @@ const Rating = styled(
         <RatingLabel>{capitalizeFirstLetter(label)}</RatingLabel>
         <ReactStars
           count={5}
-          char={'●'}
```

Comment:
I'm pretty sure there's no shortage of circle-looking Unicode characters. Have we tried using a different one? e.g. what if we use the large circle on all platforms?

Reply:
The large circle looks ridiculously large on Mac. Maybe too large on other platforms too.

From what I can tell, there are only two circle characters, "small black circle" and "large black circle", which can be recoloured. I might be wrong though. 

There is a "medium black circle" but it's hardcoded to be black and the colour can't be changed.